Alignment OpenID4VCI 1.0 (#874)
italia/eidas-it-wallet-docs: docs/en/credential-issuer-metadata.rst

Below is a non-normative example.

.. code-block:: http

    GET /.well-known/openid-credential-issuer HTTP/1.1
    Host: issuer.example.com
    Accept: application/json
    Accept-Language: it-IT, it;q=0.9

The Credential Issuer MUST respond with HTTP Status Code 200 and return the Credential Issuer Metadata containing the parameters defined in :ref:`credential-issuer-metadata:Metadata for openid_credential_issuer` within in an unsigned JSON document using the media type *application/json*.

The ``authorization_servers`` entries of the Credential Issuer Metadata can be used to obtain the Authorization Server metadata from the Oauth Authorization Server */.well-known/oauth-authorization-server* as defined in Section 3 of :rfc:`8414`. In case of ``authorization_servers`` parameter is omitted, the Credential Issuer's identifier can be used to retrieve the Authorization Server metadata.

Below is a non-normative example.

.. code-block:: http

>     GET .well-known/oauth-authorization-server HTTP/1.1

Reviewer: ```suggestion
    GET /.well-known/oauth-authorization-server HTTP/1.1
```

    Host: oauth-authorization-server.example.com

> The Oauth Authorization Server MUST respond with HTTP Status Code 200 and return the Oauth Authorization Server Metadata containing the parameters defined in :ref:`credential-issuer-metadata:Metadata for openid_credential_issuer` within in a JSON document using the media type *application/json*.

Reviewer: ```suggestion
    GET .well-known/oauth-authorization-server HTTP/1.1
    Host: oauth-authorization-server.example.com

The OAuth Authorization Server MUST respond with HTTP Status Code 200 and return the OAuth Authorization Server Metadata containing the parameters defined in :ref:`credential-issuer-metadata:Metadata for oauth_authorization_server` within in a JSON document using the media type *application/json*.
```

Reviewer 2: @giadas is that one ready to be resolved?

> The Oauth Authorization Server MUST respond with HTTP Status Code 200 and return the Oauth Authorization Server Metadata containing the parameters defined in :ref:`credential-issuer-metadata:Metadata for openid_credential_issuer` within in a JSON document using the media type *application/json*.

Author: ```suggestion
The Oauth Authorization Server MUST respond with HTTP Status Code 200 and return the Oauth Authorization Server Metadata containing the parameters defined in :ref:`credential-issuer-metadata:Metadata for oauth_authorization_server` within in a JSON document using the media type *application/json*.
```

  * - **credential_hash_alg_supported**
    - The supported algorithm used by the Wallet Instance to hash the Digital Credential for which the Status Assertion is requested. It is RECOMMENDED to use *sha-256*. (See `OAUTH-STATUS-ASSERTION`_ Section 11.1.).
  * - **batch_credential_issuance**

>     - Object containing information about the Credential Issuer's support for issuance of Credentials in a batch at the Credential Endpoint. The presence of this parameter means that the Credential issuer supports more than one key proof in the ``proofs`` parameter in the Credential request so can issue more than one Digital Credential for the same Credential with the same attributes about the Holder in a single request/response. The parameter that MUST be included is:

Reviewer: ```suggestion
    - Object containing information about the Credential Issuer's support for issuance of Credentials in a batch at the Credential Endpoint. The presence of this parameter means that the Credential Issuer supports more than one key proof in the ``proofs`` parameter in the Credential request so can issue more than one Digital Credential for the same Credential with the same attributes about the Holder in a single request/response. The parameter that MUST be included is:
```
